.github/workflows/autobuild.yml

```diff
@@ -212,9 +217,27 @@ jobs:
           retention-days:           31
           if-no-files-found:        error # 'warn' or 'ignore' are also available, defaults to `warn`
 
+      # Notarize macOS build, if needed
+      - name: "Notarize macOS Release Build"
+        if: ${{ (steps.step_cmd3_postbuild.outputs.artifact_1 != '') && (steps.step_macos_build.outputs.macos_signed == 'true') && contains(needs.create_release.outputs.publish_to_release, 'true') }}
+        id: notarize-macOS-app
+        uses: devbotsxyz/xcode-notarize@v1
```

Comment:
Can you pin this to a specific commit?

Reply:
Dunno yet. My plan is to build the official release branch when it’s ready

Reply:
> Can you pin this to a specific commit?

Did you mean to a specific commit of `devbotsxyz/xcode-notarize@v1`? Is there an advantage to doing so?